unknown_jam_store.go

```diff
@@ -18,10 +18,17 @@ func (store *UnknownJamStore)StoreJam(jamText string)  {
 	datastore.Put(store.Context, key, &UnknownJam{JamText: jamText})
 }
 
-func (store *UnknownJamStore)JamInStore(jamText string) bool  {
+func (store *UnknownJamStore)JamInStore(jamText string) *datastore.Key  {
```

Comment:
should this be named getUnkownJamKey or something like that?

Reply:
I considered that but it only gives the key and not the data.

Reply:
lookupJamKey?